doc/api/process.md

```diff
@@ -1773,18 +1773,17 @@ tarball.
 * `lts` {string} a string label identifying the [LTS][] label for this release.
   This property only exists for LTS releases and is `undefined` for all other
   release types, including _Current_ releases. Currently the valid values are:
```

Comment:
If we're not going to maintain a list of all possible values, then I think we should change the preceding wording here from "Currently the valid values are:" to something like "Valid values may include:" to indicate that the following list is a subset. While "Argon" and others may no longer be *supported* they are still technically LTS release lines that someone could encounter.

Reply:
How about "Valid values include the LTS Release Codenames (including those that are no longer supported). A non-exhaustive example of these codenames include:"

Reply:
Perhaps the links to [Release schedule](https://nodejs.org/en/about/releases/) and [Changelogs on Github](https://github.com/nodejs/node/tree/master/doc/changelogs) should also be included?

Reply:
@schamberg97 That sounds good, feel free to add those :slightly_smiling_face:

Reply:
I'll do it, but I am away from computer until midgnight

Reply:
We currently have a waiting period of 48 hours before we merge any non-trivial PR anyway, so take your time :)

Reply:
Sorry for the delay, had problems with my personal OSS project that was a bigger priority for me. This should probably be reviewed again, just in case